packages/typography/src/Error/Error.styles.ts

```diff
@@ -8,33 +8,33 @@ import {
   typeScales,
 } from '@leafygreen-ui/tokens';
 
-export const errorMessageStyle = css`
-  font-family: ${fontFamilies.default};
-  font-weight: ${fontWeights.regular};
-  font-size: inherit;
-  line-height: inherit;
+export const getErrorMessageStyle = ({
+  theme,
+  baseFontSize,
+}: {
+  theme: Theme;
+  baseFontSize: BaseFontSize;
+}) => {
+  const fontSize =
+    baseFontSize === BaseFontSize.Body1
+      ? typeScales.body1.fontSize
+      : typeScales.body2.fontSize;
+  const lineHeight =
+    baseFontSize === BaseFontSize.Body1 ? typeScales.body1.lineHeight : 20;
 
-  // Unsets browser default margins
-  margin-block-start: 0;
-  margin-block-end: 0;
-`;
+  return css`
+    font-family: ${fontFamilies.default};
+    font-weight: ${fontWeights.regular};
+    font-size: inherit;
+    line-height: inherit;
```

Comment:
is there still a need to include these `inherit` values if we set these attributes further down?